nengo/utils/tests/test_stdlib.py

```diff
@@ -47,8 +47,8 @@ def __call__(self, a, b):
 
     assert checked_call(np.sin) == (None, False)
     assert checked_call(np.sin, 0) == (0, True)
-    assert checked_call(np.sin, 0, np.array([1])) == (np.array([0]), True)
-    assert checked_call(np.sin, 0, np.array([1]), 1) == (None, False)
+    assert checked_call(np.sin, 0, np.array([1.])) == (np.array([0]), True)
```

Comment:
For some reason that didn't throw an error, but to be consistent I'll change it in the next commit.